pages/discover.js

```diff
@@ -30,12 +44,24 @@ export default function Popular({ data }) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <Page>
-        <h1 className="text-4xl mb-4 font-bold">Discover LinkFree Profiles</h1>
+        <h1 className="text-2xl md:text-4xl mb-4 font-bold">
+          Discover LinkFree Profiles
+        </h1>
 
         <div className="mb-12">
-          <h2 className="text-xl font-bold mb-4">Random LinkFree Profiles</h2>
+          <div className="flex flex-col md:flex-row gap-3 items-center justify-between mb-4">
+            <h2 className="text-md md:text-xl font-bold">
+              Random LinkFree Profiles
+            </h2>
+            <button
+              className=" text-orange-600 border border-gray-300 font-bold hover:border-orange-600 py-2 text-md md:text-lg px-3 rounded-md "
+              onClick={() => refreshProfiles()}
+            >
+              View More
```

Comment:
I think the word `reload` or `refresh` would be better, I feel with `view more` it feels like it will take me to another page with a bigger list of that type